runner/runner/handlers.py

```diff
@@ -33,15 +31,13 @@ def __init__(self):
 )
 def pull_image(self, task) -> None:
     package = task.get("package")
-    fqi = f"{self.REGISTRY}/{package}"
-    self._docker.images.pull(fqi)
+    self._docker.images.pull(package)
 
-    logger.debug(f"Pulled {fqi}")
-    return task
+    logger.debug(f"Pulled {package}")
 
 
 @app.task(base=PackageClient, bind=True)
-def run_task(self, task):
+def run_task(self, _=None, task=None):
```

Comment:
I sort of traded one annoyance (pull_image returning the task only for the benefit of the celery chaining) for another here (having an unused `_` parameter so that the celery chaining ends up with the arguments properly aligned), but I felt like this annoys me ever so slightly less.